src/js/controller/controller.js

```diff
@@ -847,8 +851,6 @@ Object.assign(Controller.prototype, {
             try {
                 setPlaylist(_model, playlist, feedData);
             } catch (error) {
-                _model.set('item', 0);
-                _model.set('playlistItem', null);
```

Comment:
This causes the player to dispatch a "playlstItem" event with a value of `null` as it tries to go into a `no playable sources` error state (ex: `jwplayer().load([])`). That breaks many things and prevents the player from getting to the user error.